images/ingress-in-cluster-nlb-local.svg

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Typo in this diagram, it lists Pod B twice. 

The left-most Pod B (for Service 1) should be Pod A.